lib/engine/auto_router.rb

```diff
@@ -288,130 +270,129 @@ def js_evaluate_combos(_rb_sorted_routes, _route_timeout)
       conflicts = 0
       now = Time.now
 
-      %x(
```

Comment:
It seems like only spacing changed.  Is there a way to suppress spacing diffs in the compare?  Or at least see them side by side?

Reply:
corrext